src/tests/ftest/util/dfuse_utils.py

```diff
@@ -30,7 +30,6 @@ def __init__(self, namespace, command, path=""):
         self.sys_name = FormattedParameter("--sys-name {}")
         self.thread_count = FormattedParameter("--thread-count {}")
         self.eq_count = FormattedParameter("--eq-count {}")
-        self.singlethreaded = FormattedParameter("--singlethread", False)
```

Comment:
Luckily no tests are using this

Reply:
I did check, although only by grepping.

Reply:
Searching for `singlethreaded` is 99% sufficient. We don't know if someone somewhere has a different config that uses this, but it's pretty unlikely